### ISSUE TITLE
Residence Error

### ISSUE THREAD

**Issue by MartinPPG** [PEXPlugins/PermissionsEx#2792]
Please HELP!, I have HEXXIT server with PEX and Residence, and when I use /residence (alias: /res) it wrote: "An internal error occured while attempting to perform this command." Running Hexxit server on version: 1.5.2. Thanks

**Comment by Stormbow** [comment]
I'm not promising I can solve this problem, since you're using a version of Minecraft which is _literally_ more than 4 years old, but I'll take a shot at it...  (Minecraft 1.5.2 was released May 2nd, 2013.)

Copy the full server startup log— from the moment the server starts until text stops printing to the server console and you see a message similar to "Done!" —to [Pastebin](http://pastebin.com) and leave the link generated by that site in this thread.

Do the same with your permissions.yml file, which will be located inside the plugins\PermissionsEx folder.  The formatting of this file (for you) is going to be unlike the formatting of any permissions.yml file I used on a server because I didn't start _running_ a server of my own until 1.6.4 and I don't have any files previous to 1.7.2 on this computer, but I _may_ be able to help if there are fundamental errors in your file.

I'll see what I can figure out.
